content/docs.pl.md

```diff
@@ -0,0 +1,184 @@
+# Do not translate or change CHANGELOG or ACKNOWLEDGEMENTS.
+# Translate the other words below, including "Date:" and "Editor:"
+# Translate the Working Group name. Leave the Working Group acronym in English.
+# Do not change the dates in the footer below.
+footer: >
+   <p><strong>Data:</strong> Aktualizacja 14 marca 2024. Pierwsza publikacja w czerwcu 2005.</p>
+   <p><strong>Redaktor:</strong> <a href="https://www.w3.org/People/Shawn/">Shawn Lawton Henry</a>. Współpraca: <a href="https://www.w3.org/People/hidde/">Hidde de Vries</a> i <a href="https://www.w3.org/People/shadi/">Shadi Abou-Zahra</a>.</p>
+   <p>Opracowane z udziałem Grupy Roboczej ds. Edukacji i Promocji (<a href="https://www.w3.org/WAI/about/groups/eowg/">EOWG</a>).</p>
+
+---
+
+{::nomarkdown}
+{% include box.html type="start" h="2" title="Podsumowanie" class="full" %}
+{:/}
+
+Ta strona przedstawia różne dokumenty pomocnicze (WCAG) 2 i dodatkowe wytyczne.  Pomaga się zorientować, gdzie szukać odpowiednich informacji.
+
+{::nomarkdown}
+{% include box.html type="end" %}
+{:/}
+
+{::nomarkdown}
+{% include_cached toc.html type="start" title="Treść strony" class="simple" %}
+{:/}
+
+{::options toc_levels="2,3,4" /}
+
+-   This text will be replaced by the TOC.
+{:toc}
+
+
+{::nomarkdown}
+{% include_cached toc.html type="end" %}
+{:/}
+
+## Standard WCAG 2
+
+Wytyczne dla dostępności treści internetowych (WCAG) 2 to stabilny, referencyjny standard techniczny. Dowiedz się więcej w artykule [[Omówienie WCAG 2]](/standards-guidelines/wcag/).
+
+Podczas tworzenia treści internetowych i narzędzi internetowych większość osób będzie korzystać z dokumentów przedstawionych poniżej, zamiast dokumentu z rzeczywistymi standardami technicznymi.
+
+## Króki przewodnik / Jak spełnić WCAG 2 / Lista kontrolna WCAG 2
+
+**Kluczowym zasobem dla projektantów i programistów korzystających z WCAG 2 jest [Jak spełnić WCAG 2 (Krótki przewodnik): Konfigurowalne szybkie odniesienie do wymagań WCAG 2 (kryteria sukcesu) i technik](https://www.w3.org/WAI/WCAG22/quickref/).** Zawiera wszystkie wytyczne WCAG 2 i kryteria sukcesu. Zasadniczo jest to lista kontrolna WCAG 2.
+
+Możesz dostosować Jak spełnić WCAG 2, aby pokazać, co Cię interesuje w tym czasie. Istnieją „filtry” dla różnych ról, tematów i technologii (takich jak HTML, CSS, ARIA). Możesz także wybrać wyświetlanie [kryteriów sukcesu na poziomie A, AA lub AAA](https://www.w3.org/WAI/WCAG22/Understanding/conformance#levels)).
+
+## Dokumenty pomocnicze
+
+Dokumenty pomocnicze odnoszą się bezpośrednio do wytycznych WCAG i kryteriów sukcesu. Te dokumenty **nie są wymagane** do spełnienia WCAG; są „informacyjne” lub inaczej „nienormatywne”.  
+
+### Objaśnienia
+
+<cite>Objaśnienia WCAG</cite> to przewodnik pomagający zrozumieć i wdrożyć WCAG.  Zawiera dodatkowe informacje dla osób, które chcą dokładniej zrozumieć wytyczne i kryteria sukcesu. Przedstawia on:
+
+- intencje
+- jak wymagania WCAG pomagają osobom z różnymi niepełnosprawnościami
```

Comment:
Original text:
> how it helps people with different disabilities

Polish translation (automatically re-translated into English):
> How WCAG requirements help people with different disabilities

This also applies to success criteria. Would you think of a translation closer to the original text?